bundles/org.openhab.binding.mikrotik/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="mikrotik" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:binding="https://openhab.org/schemas/binding/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/binding/v1.0.0 https://openhab.org/schemas/binding-1.0.0.xsd">
+
+	<name>Mikrotik Binding</name>
+	<description>
+		This is the binding for integrating Mikrotik RouterOS powered devices (routers, access points, switches,
+		etc) to facilitate WiFi clients and network interface tracking.
+	</description>
+	<author>Oleg Vivtash</author>
```

Comment:
The author tag is deprecated and should therefore be removed. See openhab/openhab-core#1844.